workflow/airflow_libero/docker-compose-celery-executor.yml

```diff
@@ -0,0 +1,85 @@
+version: '3'
+services:
+    redis:
+        image: 'redis:3.2.7'
+
+    postgres:
+        image: postgres:9.6
+        environment:
+            - POSTGRES_USER=airflow
+            - POSTGRES_PASSWORD=airflow
+            - POSTGRES_DB=airflow
+            - PGDATA=/var/lib/postgresql/data
+        volumes:
+            - 'postgres:/var/lib/postgresql/data'
+
+    rabbitmq: # ONLY USED AS A DUMMY EVENT BUS!
+        image: 'rabbitmq:3.7.3-management'
+        ports:
+            - '8090:15672'
+            - '5672:5672'
+        volumes:
+            - 'rabbitmq:/var/lib/rabbitmq'
+
+    webserver:
+        build:
+          context: .
+        depends_on:
+            - postgres
+            - redis
+        environment:
+            - LOAD_EX=n
+            - FERNET_KEY=46BKJoQYlPPOexq0OhDZnIlNepKFf87WFwLbfzqDDho=
+            - EXECUTOR=Celery
+        volumes:
+            - './dags:/usr/local/airflow/dags'
+            - './plugins:/usr/local/airflow/plugins'
+        ports:
+            - "8080:8080"
+        command: webserver
+        healthcheck:
```

Comment:
nice healthcheck, but it can be moved to the Dockerfile with `HEALTHCHECK` so that it is reusable in any docker-compose configuration?